Description: Disable feature attr-macro, dep proptest-macro not yet packaged
Last-Update: 2024-06-30
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -98,5 +98,5 @@
 
-[dependencies.proptest-macro]
-version = "0.1"
-optional = true
+#[dependencies.proptest-macro]
+#version = "0.1"
+#optional = true
 
@@ -143,3 +143,3 @@
 atomic64bit = []
-attr-macro = ["proptest-macro"]
+#attr-macro = ["proptest-macro"]
 bit-set = [
